internal/netdev/netdev.go

```diff
@@ -159,7 +177,7 @@ func listInterfaces() ([]string, error) {
 // @result = 2 => Interface is down
 // @result = 3 => Interface is unknown or state of the interface is unknown for some reason
 func getInterfaceState(data *IfaceData) error {
-	basePath := "/sys/class/net/" + data.Name
+	basePath := netDevicePath + data.Name
```

Comment:
Maybe we can use https://pkg.go.dev/path#Join

Reply:
done